packages/sw-broadcast-cache-update/test/sw/behavior.js

```diff
@@ -0,0 +1,55 @@
+importScripts(
+  '/node_modules/mocha/mocha.js',
+  '/node_modules/chai/chai.js',
+  '/node_modules/sw-testing-helpers/build/browser/mocha-utils.js',
+  '/packages/sw-broadcast-cache-update/build/sw-broadcast-cache-update.min.js'
+);
+
+const expect = self.chai.expect;
+mocha.setup({
+  ui: 'bdd',
+  reporter: null,
+});
+
+describe('Test of the Behavior class', function() {
+  const channelName = 'test-channel';
+  const headersToCheck = ['one', 'two'];
+  const source = 'test-source';
+
+  it(`should throw when Behavior() is called without any parameters`, function() {
+    expect(() => new goog.broadcastCacheUpdate.Behavior()).to.throw();
```

Comment:
Sure thing. I've been thinking more about https://github.com/GoogleChrome/sw-helpers/issues/98 and I think we can end up in a place where we have non-generic, contextual `Error`s without all the overhead of configuring an instance of an `ErrorFactory`. But for now I'll go that route.